.github/workflows/ci.yml

```diff
@@ -8,36 +8,27 @@ on:
   pull_request:
 
 jobs:
-  build_ruby:
+  build-ruby:
     runs-on: ubuntu-latest
     strategy: 
       matrix: 
-        ruby-version: [ "2.0.0-p648", "2.1.10", "2.2.10", "2.3.8", "2.4.10", "2.5.8", "2.6.6", "2.7.1", "jruby-9.2.12.0" ]
+        ruby-version: [ 2.0.0-p648, 2.1.10, 2.2.10, 2.3.8, 2.4.10, 2.5.8, 2.6.6, 2.7.1, jruby-9.2.12.0 ]
     steps:
       - uses: actions/checkout@v2
 
-      - uses: actions/cache@v2
-        id: ruby-cache
-        with:
-          path: ~/.rubies/ruby-${{ matrix.ruby-version }}
-          key: v5-ruby-cache-${{ matrix.ruby-version }}
-          restore-keys: |
-            v5-ruby-cache-${{ matrix.ruby-version }}
-
       - name: Build Ruby ${{ matrix.ruby-version }}
-        id: build-ruby
         uses: ./.github/actions/build-ruby
         with:
           ruby-version: ${{ matrix.ruby-version }}
   
-  unit_test:
-    needs: build_ruby
+  mini-env:
+    needs: build-ruby
     runs-on: ubuntu-latest
     strategy: 
       fail-fast: false
       matrix: 
-        ruby-version: [ "2.0.0-p648", "2.1.10", "2.2.10", "2.3.8", "2.4.10", "2.5.8", "2.6.6", "2.7.1", "jruby-9.2.12.0" ]
-        rails: [norails, rails60, rails52, rails51, rails42, rails41, rails40, rails32, rails31, rails30]
+        ruby-version: [ 2.0.0-p648, 2.1.10, 2.2.10, 2.3.8, 2.4.10, 2.5.8, 2.6.6, 2.7.1, jruby-9.2.12.0 ]
+        rails: [ norails, rails60, rails52, rails51, rails42, rails41, rails40, rails32, rails31, rails30 ]
```

Comment:
Similar to my other comment, we should swap the order of these so its grouped by rails version

Reply:
done!